entries/e/earthclassmail.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Earth Class Mail": {
+    "domain": "earthclassmail.com",
+    "img": "earthclassmail.com.png",
+    "url": "https://www.earthclassmail.com/",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "documentation": "https://help.earthclassmail.com/hc/en-us/articles/4410499059351-How-to-enable-Two-Factor-Authentication-in-Earth-Class-Mail",
```

Comment:
```suggestion
    "documentation": "https://help.earthclassmail.com/hc/en-us/articles/4410499059351",
```

Reply:
Thanks for the review.  I did look for an SVG, but I bow to your search-fu.